youtube_dl/extractor/piksel.py

```diff
@@ -113,11 +101,25 @@ def _real_extract(self, url):
             })
         self._sort_formats(formats)
 
+        subtitles = {}
+        for caption in video_data.get('captions', []):
+            caption_url = caption.get('url')
+            caption_format = caption.get('url').split(' ')[-1].split('.')[-1]
+            if not caption_url or 'vtt' not in caption_format:
+                continue
+            subtitles.setdefault(caption.get('locale', 'en'), []).append({
+                'url': caption_url,
+                'ext': {
+                    'webvtt': 'vtt',
```

Comment:
the ext is present in the URL, so there is no need for this part, the extension will be extracted later by common code.

Reply:
agreed. will address as suggested.